warehouse/models/staging/payments/elavon/int_elavon__billing_transactions_deduped.sql

```diff
@@ -0,0 +1,79 @@
+{{ config(materialized='table') }}
+
+WITH
+
+stg_elavon__transactions AS (
```

Comment:
For conciseness, could just combine this with the following CTE (`SELECT * WHERE batch type`) and same with deposits